docs/spec/v1beta2/helmcharts.md

```diff
@@ -202,6 +202,16 @@ spec:
 Values files also affect the generated artifact revision, see
 [artifact](#artifact).
 
+### Missing values files
```

Comment:
```suggestion
### Ignore missing values files
```

Reply:
I'd left off the "Ignore" word as I thought this also nicely described the default behaviour: considering a missing values file to be an error, but I'm happy either way :-)

Reply:
The title name must match the spec field.

Reply:
It's updated to do so.